.github/workflows/src/set-status.js

```diff
@@ -82,12 +82,25 @@ export async function setStatusImpl({
     issue_number: issue_number,
     per_page: PER_PAGE_MAX,
   });
-  const overridingLabels = labels.map((label) => label.name);
+  const prLabels = labels.map((label) => label.name);
 
-  core.info(`Labels: ${overridingLabels}`);
+  core.info(`Labels: ${prLabels}`);
 
-  if (overridingLabels.includes(overridingLabel)) {
-    const description = `Found label '${overridingLabel}'`;
+  // Parse overriding labels (comma-separated string to array)
+  const overridingLabelsArray = overridingLabel
+    ? overridingLabel
+        .split(",")
+        .map((label) => label.trim())
+        .filter((label) => label)
```

Comment:
```suggestion
```

If this is to remove empty labels, please add a comment explaining it.

Reply:
added comment.